### ISSUE TITLE
Uncaught ReferenceError: toggleComments is not defined

### ISSUE THREAD

**Issue by nrkhwaja** [javaguirre/comment#45]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.15.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [comment](https://github.com/havber/comment) package 0.5.0

### Stack Trace

Uncaught ReferenceError: toggleComments is not defined

```
At /Users/khwajanr/.atom/packages/comment/lib/comment-utils.js:50

ReferenceError: toggleComments is not defined
    at Object.toggleHtmlSingleLineOrEmptySelection (/packages/comment/lib/comment-utils.js:50:44)
    at Object.toggleSingleCommentsWithRange (/packages/comment/lib/comment-utils.js:33:18)
    at Object.toggle (/packages/comment/lib/comment-utils.js:13:18)
    at /packages/comment/lib/comment.js:31:15)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -3:06.8.0 settings-view:uninstall-packages (input.hidden-input)
     -3:03.4.0 core:backspace (input.hidden-input)
  3x -2:47 core:confirm (ul.list-inline.tab-bar.inset-panel)
  3x -2:45.5.0 editor:newline (input.hidden-input)
     -2:41.3.0 core:backspace (input.hidden-input)
     -2:39 insert-timestamp:DateT (input.hidden-input)
  2x -2:37.1.0 editor:newline (input.hidden-input)
     -2:35.6.0 insert-timestamp:DateT (input.hidden-input)
     -2:31.7.0 core:backspace (input.hidden-input)
     -2:22.7.0 settings-view:install-packages-and-themes (input.hidden-input)
  2x -1:38.2.0 editor:newline (input.hidden-input)
     -1:32.6.0 comment-timestamp:comment (input.hidden-input)
     -1:30.7.0 editor:newline (input.hidden-input)
     -1:01.1.0 key-binding-resolver:toggle (input.hidden-input)
     -0:38.8.0 git-diff:toggle-diff-list (input.hidden-input)
     -0:26 comment:toggle (input.hidden-input)
```

### Non-Core Packages

```
comment 0.5.0 
comment-timestamp 0.3.0 
insert-timestamp 0.2.4 
language-markdown 0.20.0 
```

**Comment by javaguirre** [comment]
Fixed on v0.6.0, thanks for reporting!

**Comment by nrkhwaja** [comment]
verified working. 👍
